src/lib/nlcManager.js

```diff
@@ -288,12 +288,17 @@ NLCManager.prototype._getClassifier = function(){
 				});
 
 				if (filteredClassifiers.length < 1){
-					// no classifiers found by this name, so create one and start training.
-					this._startTraining().then((result) => {
-						dfd.resolve(result);
-					}).catch((err) => {
-						dfd.reject(err);
-					});
+					if (doNotTrain) {
+						dfd.reject('No classifier found by this name.');
```

Comment:
Let's add the name of the classifier on the error message, with the crawler and objectstore work we are moving towards having multiple classifiers.